### PR TITLE
transactions page

### PR DESCRIPTION
**Pull request by Soimil**

### CODE REVIEW

File: apps/ui/lib/ui_web/live/transactions_live.ex [hunk 1]
```diff
@@ -0,0 +1,160 @@
...
+        </div>
+      </div>
+      <% else %>
+        You are not have any transactions
+      <% end %>
+    """
+  end
+
+  def mount(%{}, socket) do
+    IO.inspect(transactions_data(50, 1, ""))
```
Comment (Cinderella-Man) [inline apps/ui/lib/ui_web/live/transactions_live.ex]: remove this line please

File: apps/hefty/lib/hefty.ex [hunk 2]
```diff
@@ -179,4 +179,25 @@ defmodule Hefty do
     )
     |> Hefty.Repo.one()
   end
+
+  def fetch_transactions(offset, limit, symbol \\ "") do
+    Logger.debug("Fetching transactions for a symbol", symbol: symbol)
```
Comment (Cinderella-Man) [inline apps/hefty/lib/hefty.ex]: could you please stop using metadata for data that you would like to include in the message - it was a mistake and should be fixed wherever happened before - it should now look like:
```
Logger.debug("Fetching transactions for a symbol(#{symbol})")
```

**Review by Cinderella-Man — CHANGES_REQUESTED** [review]
Looks good beside those minor changes - could you please fix and push again?

File: apps/hefty/lib/hefty.ex (hunk 2 above)
Comment (Soimil) [reply apps/hefty/lib/hefty.ex]: Updated.

**Pull request merged**